assets/js/form.js

```diff
@@ -0,0 +1,45 @@
+// whole form
+// view
+var FormView = Backbone.View.extend({
+  template: AppTemplates.form,
+
+  el: '#target',
+
+  initialize: function() {
+    this.render();
+  },
+
+  events: {
+    'submit form': 'createPerson'
+
+    // 'form .submit': 'submitForm'
+  },
+
+  createPerson: function(ev) {
+    ev.preventDefault();
+
+    var firstName = this.$('#first-name').val();
```

Comment:
You are looking for HTML elements within your view that are `id` but look at your template, you have classes on your inputs.